modules/pel/peltool/peltool.py

```diff
@@ -335,7 +335,18 @@ def listOption(path: str, config: Config, extension: str, rev: bool =False):
             file_list.append(file)  ## create list of file names
         # Only process top level directory
         break
-    file_list.sort(reverse=rev) 
+    file_list.sort(reverse=rev)
+    if all:
+        final_string = "[ \n"
+        for file in file_list:
+            with open(os.path.join(root, file), 'rb') as fd:
+                data = fd.read()
+                stream = DataStream(data, byte_order='big', is_signed=False)
+                _, json_string = parsePEL(stream, config, False)
+                final_string = final_string + json_string + ",\n"
```

Comment:
I think this would put a command on the very last PEL, which JSON considers invalid syntax.

Reply:
The string slicing `allPELsData[:-2]` takes care of the extra comma in the last line.

Reply:
but it doesn't seem to work?
```
        ]
    }
},

]

```